RadarSDK/Radar.m

```diff
@@ -640,6 +660,12 @@ + (NSString *)stringForMode:(RadarRouteMode)mode {
     case RadarRouteModeCar:
         str = @"car";
         break;
+    case RadarRouteModeTruck:
+        str = @"truck";
+        break;
+    case RadarRouteModeMotorScooter:
+        str = @"motor_scooter";
```

Comment:
right now this is `motorbike` in radar (which gets mapped to `motor_scooter` in valhalla)

https://github.com/radarlabs/server/pull/3118/files#diff-02d01ebe7d564e3772bd83bab5289d376d6c0e5781f9fea7be0d4e6156a579d9R169

https://valhalla.readthedocs.io/en/latest/api/turn-by-turn/api-reference/#costing-models

Reply:
Good catch, will swap